### ISSUE TITLE
About the number of images in SfM.

### ISSUE THREAD

**Issue by theFool32** [filipradenovic/cnnimageretrieval-pytorch#12]
Hi, I check the `cdis` in pkl file, and find 
```
len(train['cids']) == 91642
len(val['cids']) == 6403
```
Does it mean training dataset only uses 91k images and val dataset use 6k images, not 120k or 30k.

Thanks.

**Comment by filipradenovic** [comment]
Hello, the names are historical, ie, retrieval-SfM-30k was used in our [ECCV16 paper](http://cmp.felk.cvut.cz/~radenfil/publications/Radenovic-ECCV16.pdf), and it has 22156 training and 6403 validation images, which constitutes ~30k in total. For retrieval-SfM-120k, we increase the size of both training and validation set with a factor of 4, approximately, hense the name 120k. However, in the end, using old (smaller) validation set of 6403 images is enough and faster to evaluate, and that is the reason for 91642 and 6403 images.

Finally, if you want to learn whitening with retrieval-SfM-30k-whiten or retrieval-SfM-120k-whiten you will indeed use all 30k or 120k images, respectively.

I hope this clears things up.

**Comment by theFool32** [comment]
Got it. Thanks a lot.
